### ISSUE TITLE
What is the current landscape of mobile RSS apps?

### ISSUE THREAD

**Issue by swanson** [swanson/stringer#423]
Stringer emulates the Fever API in order to be compatible with some popular (circa-2013) RSS mobile apps.

What are the current popular apps? Do they support using Stringer as your "RSS Source"? Is there a different API that we should emulate? If so, what clients will that gives us access to?

(Personally, I only read on desktop -- so please weigh in if you are a mobile RSS user)

**Comment by queuebit** [comment]
I still use it occasionally on iOS via [Unread](http://supertop.co/unread/) which still has Fever support. Unread isn't one of the "current popular apps" though.

**Comment by pascalw** [comment]
I'm using Reeder on both iOS and Mac and that works great with the Fever API. I don't know about other iOS apps that support Fever.

I've used an Android phone for a while though and there the only option seemed to be [Press](https://play.google.com/store/apps/details?id=com.twentyfivesquares.press&hl=nl) which worked OK, but feels a bit outdated. Looking at other available apps it seems that [some](https://play.google.com/store/apps/details?id=com.noinnion.android.greader.readerpro) support TinyTinyRSS (presumable via it's [JSON API](https://tt-rss.org/gitlab/fox/tt-rss/wikis/ApiReference)).

So perhaps emulating the TinyTinyRSS would expand options a bit.

**Comment by jeroenj** [comment]
I've been using Reeder on iOS and OSX too. I think it's still one of the most popular RSS apps on both platforms.

**Comment by lorentrogers** [comment]
There's no good app that supports Fever on F-Droid, which is too bad. I really wish I could access this on my phone, but desktop works ok for now.

**Comment by jackivan88** [comment]
Press used to be the top app for me, but it seems to be abandoned.

**Comment by pascalw** [comment]
Unfortunately on Android there's not much choice anymore. Most feed reader apps don't support any self hosted feed readers at all, only Feedly, Feedbin etc.

For myself I'm now using a patched version of [Newsfold](https://play.google.com/store/apps/details?id=it.mvilla.android.quote) with a hacked up implementation of the Inoreader API in Stringer. That works pretty well but is not ideal because it requires patching the APK.

**Comment by gabrielpoca** [comment]
I'm closing this issue for lack of activity.

**Comment by romanar** [comment]
did anyone found any good android application that works with Fever API?

**Comment by pascalw** [comment]
I'm still using patched Newsfold with an Inoreader API for Stringer. If there's interest I can publish the sources for this. It's not very polished but works well.

**Comment by romanar** [comment]
@pascalw That would be great.

**Comment by pascalw** [comment]
@romanar ok here it is.

1. Apply [this commit](https://github.com/pascalw/stringer/commit/bad7da5a9f79cfac90957995e34aeb177c4da4d7) to your Stringer instance.
2. Add a `INOREADER_TOKEN=somethingRandomHere` environment variable to Stringer
3. Get a copy of Newsfold apk. Note I've only tested this with Newsfold 1.2.
4. Run [this patch script](https://gist.github.com/pascalw/2b85f7dbc2f8149bedc9ce316882b685) like so `./patch.sh ./newsfold.apk https://stringer.example.com/inoreader/INOREADER_TOKEN_HERE/"
5. Install the APK on your phone, choose Inoreader and presto!

Couple of things to note:

1. The code is very messy. I wrote it almost 1.5 years ago just to get something working and haven't touched it ever since. It works absolutely fine.
2. Security wise it relies on the `INOREADER_TOKEN` you specify.
3. Use at your own risk ;-)
